deploy/src/Microsoft.Azure.IIoT.Deployment/Configuration/HelmSettings.cs

```diff
@@ -13,7 +13,7 @@ class HelmSettings {
         // Defaults
         public static string _defaultRepoUrl = "https://microsoft.github.io/charts/repo";
         public static string _defaultChartVersion = "0.3.2";
-        public static string _defaultImageTag = "2.7.206";
```

Comment:
Please revert this back to `2.7.206`. Updating IAI for `2.8.1` release entails not only change of image tag and chart version but also cleanup of resources that were deployed for microservices that are removed from the deployment. And we have a separate PBI for doing all of that.

Reply:
reverted